packages/common/src/content/compose.ts

```diff
@@ -40,16 +41,18 @@ const getOnlyQueryLayer = (layers: ILayerDefinition[]) => {
 };
 
 const shouldUseLayerInfo = (
+  item: IItem,
   layer: Partial<ILayerDefinition>,
   layers: Array<Partial<ILayerDefinition>>,
-  url: string
+  requestOptions?: IHubRequestOptions
 ) => {
   return (
+    !isProxiedCSV(item, requestOptions) &&
```

Comment:
@tomwayson I made this change here because as far as I can tell, we should never use layer info for proxied csvs (affects `name` and `description`)

Reply:
yeah, I was thinking the same.